tests/test_authorisation.py

```diff
@@ -1,21 +1,22 @@
-from app.main.authorisation import get_local_authority_place_names
+from app.main.authorisation import get_local_authority_and_place_names
 
 
 def test_get_local_authority_place_names():
-    user1_places = get_local_authority_place_names("user@bolton.gov.uk")
-    user2_places = get_local_authority_place_names("user@newcastle-staffs.gov.uk")
-    user3_places = get_local_authority_place_names("user@wigan.gov.uk")
-    user4_places = get_local_authority_place_names("user@wmadeup.gov.uk")
+    user1_places = get_local_authority_and_place_names("user@bolton.gov.uk")
+    user2_places = get_local_authority_and_place_names("user@newcastle-staffs.gov.uk")
+    user3_places = get_local_authority_and_place_names("user@wigan.gov.uk")
+    user4_places = get_local_authority_and_place_names("user@wmadeup.gov.uk")
 
     assert user1_places == (
-        "Farnworth",
-        "Bolton",
+        ("Bolton Metropolitan Borough Council",),
+        (
+            "Farnworth",
+            "Bolton",
+        ),
     )
     assert user2_places == (
-        "Newcastle-Under-Lyme Town Centre",
-        "Kidsgrove",
-        "Newcastle-under-Lyme",
-        "Newcastle-under-Lyme Town Centre",
+        ("Newcastle-under-Lyme District Council",),
+        ("Newcastle-Under-Lyme Town Centre", "Kidsgrove", "Newcastle-under-Lyme", "Newcastle-under-Lyme Town Centre"),
     )
-    assert user3_places == ("Wigan",)
-    assert user4_places is None
+    assert user3_places == (("Wigan Metropolitan Borough Council",), ("Wigan",))
+    assert user4_places == (None, (None,))
```

Comment:
Think we're missing a test case for when a users full email, rather than just domain, is used to return some LAs and places